bookwyrm/views/preferences/export.py

```diff
@@ -162,7 +188,7 @@ def get(self, request):
         next_available = (
             jobs.first().created_date + timedelta(hours=hours) if not allowed else False
         )
```

Comment:
If this check exists for security/DoS reasons, it should also happen in the `post` method, otherwise a malicious actor could simply craft the request themselves.

Reply:
Ah, you're right. I wasn't really thinking about DDoS to be honest, just preventing impatient users from making requests in the UI - we might have the same issue in book exports. It needs to be in both GET and POST because we need to tell users when they are next allowed to do an export.